utils/validator_info.go

```diff
@@ -0,0 +1,233 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"reflect"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/chain4travel/magellan/cfg"
+	"github.com/chain4travel/magellan/models"
+)
+
+func PeerIndex(peers *models.PeersResponse, nodeID string) int {
+	for idx, peer := range peers.Result.Peers {
+		if peer.NodeID == nodeID {
+			return idx
+		}
+	}
+	return -1
+}
+func GetDate(unixTime string) string {
+	// Date in Unix Format
+	unixDateInt, _ := strconv.ParseInt(unixTime, 10, 64)
+	// UnixDate in Time format struct
+	dateFTime := time.Unix(unixDateInt, 0)
+	return strings.Split(dateFTime.String(), " -")[0]
+}
+func getDuration(startTime string, endTime string) string {
+	start, _ := time.Parse("2006-01-02 15:04:05", startTime)
```

Comment:
I think i don't get why we use these hardcoded dates in order to get the difference. Could you please explain?

Reply:
Time.Parse  parses a formatted string and returns the time value it represents. 
`func Parse(layout, value string) (Time, error)`
These are predefined layouts for use in Time.Format and time.Parse, each element shows by example the formatting of an element of the reference time. Only these values are recognized: 
	**Year:** "2006" "06"
	**Month:** "Jan" "January" "01" "1"
        Day of the week: "Mon" "Monday"
	**Day of the month:** "2" "_2" "02"
	Day of the year: "__2" "002"
	**Hour:** "15" "3" "03" (PM or AM)
	**Minute:** "4" "04"
	**Second:** "5" "05"
	AM/PM mark: "PM"

Reply:
Didn't know about these dates.. Wouldn't be better to use a constant instead of the hardcoded string `2006-01-02 15:04:05` as layout? For example smthg like `time.UnixDate` instead? in order to avoid confusion